live/global.ini

```
@@ -42549,16 +42549,16 @@ chat_emote_cry=cry
 chat_emote_cry1=cry1
 chat_emote_cry_text=Du bricht in Tränen aus.
 chat_emote_cs_forward=forward
-chat_emote_cs_forward_text=Nach oben verschieben.
+chat_emote_cs_forward_text=Nachrücken.
```

Was genau soll das hier bedeuten? Nachrücken ist ja eher militärisches Unterstützen oder so. 
Passt das dann für einen Emote?

---

Ist der Emote zum Nachrücken. Dieser langgestreckte Arm

---

Oder eben Forwärts. Kann man drüber streiten

---

Move up

---

Ich check das kurz im Spiel... 

/forward im Chat eingeben oder?

---

jo müsste sein

---

Emote geht bei mir nicht aber ich denke 
```suggestion
chat_emote_cs_forward_text=Vorrücken.
```
Würde besser passen :-D

---

:D ja man ^^